components/ConvertSteps/ConvertSteps.js

```diff
@@ -55,8 +61,9 @@ function ConvertSteps({ toAnj, fromAmount, toAmount, onReturnHome, steps }) {
 
   const attemptStepSigning = useCallback(
     async stepIndex => {
-      const onHashCreated = steps[stepIndex][1].hashCreated
+      const hashCreated = steps[stepIndex][1].hashCreated
       const createTx = steps[stepIndex][1].createTx
+      const transactionComplete = steps[stepIndex][1].transactionComplete
```

Comment:
Perhaps we can destructure these constants instead, since they're all coming from the same object?